articles/media-services/latest/live-event-error-codes.md

```diff
@@ -34,37 +34,34 @@ event.
 >| Description |The media description information was not received before the actual media data was delivered. |
->| Suggested solution|The LiveEvent does not receive the stream description (header or FLV tag) from the encoder. This is a protocol violation. Contact encoder vendor. |
+>| Suggested solution|The live event did not receive the stream description (header or FLV tag) from the encoder. This is a protocol violation. Please contact the encoder vendor. |
 >|**MPE_INGEST_MEDIA_QUALITIES_EXCEEDED** ||
 >| Description|The count of qualities for audio or video type exceeded the maximum allowed limit. |
 >| Suggested solution|When Live Event mode is Live Encoding, the encoder should push a single bitrate of video and audio.  Note that a redundant push from the same bitrate is allowed. Check the encoder preset or output settings to make sure it outputs a single bitrate stream. |
 >|**MPE_INGEST_BITRATE_AGGREGATED_EXCEEDED** ||
 >| Description|The total incoming bitrate in a live event or channel service exceeded the maximum allowed limit. |
 >| Suggested solution|The encoder exceeded the maximum incoming bitrate. This limit aggregates all incoming data from the contributing encoder. Check encoder preset or output settings to reduce bitrate. |
->|**MPE_RTMP_FLV_TAG_TIMESTAMP_INVALID** ||
->| Description|The timestamp for video or audio FLVTag is invalid from the RTMP encoder. |
->| Suggested solution|Deprecated. |
 >|**MPE_INGEST_FRAMERATE_EXCEEDED** ||
 >| Description|The incoming encoder ingested streams with frame rates exceeded the maximum allowed 30 fps for encoding live events/channels. |
->| Suggested solution|Check encoder preset to lower frame rate to under 36 fps. |
+>| Suggested solution|Please check the contribution encoder's settings and lower the frame rate to under 36 fps. |
 >|**MPE_INGEST_VIDEO_RESOLUTION_NOT_SUPPORTED** ||
 >| Description|The incoming encoder ingested streams exceeded the following allowed resolutions: 1920x1088 for encoding live events/channels and 4096 x 2160 for pass-through live events/channels. |
 >| Suggested solution|Check encoder preset to lower video resolution so it doesn't exceed the limit. |
 >|**MPE_INGEST_RTMP_TOO_LARGE_UNPROCESSED_FLV** |
 >| Description|The live event has received a large amount of audio data at once, or a large amount of video data without any key frames. We have disconnected the encoder to give it a chance to retry with correct data. |
->| Suggested solution|Ensure that the encoder sends a key frame for every key frame interval(GOP).  Enable settings like "Constant bitrate(CBR)" or "Align Key Frames". Sometimes, resetting the contributing encoder may help. If it doesn't help, contact encoder vendor. |
+>| Suggested solution|Ensure that the encoder sends a key frame for every key frame interval(GOP).  Enable settings like "Constant bitrate(CBR)" or "Align Key Frames". Some times, resetting the contributing encoder may help. If it doesn't help, contact encoder vendor. |
```

Comment:
This is incorrect. Sometimes is one word.  Alternately, you could just leave the word out and simply say "Resetting the contributing..."